media/icon.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg xmlns:svg="https://www.w3.org/2000/svg" xmlns="https://www.w3.org/2000/svg" width="48" height="48" id="svg2856" version="1.1"><style>.s0{fill:#2e3436;filter:url(#filter3864);opacity:0.3;}.s1{fill:#4e9a06;stroke-linecap:round;stroke:#316004;}</style><defs id="defs2858"><filter id="filter3864"><feGaussianBlur stdDeviation="0.2" id="feGaussianBlur3866"/></filter></defs><metadata id="metadata2861"/><g id="layer1"><path d="M0 16 0 28 21 40.1l21-12.4 0-11.5L20.8 4 0 16z" id="path3047" transform="translate(3.4641013,6)" style="fill:#e9b96e;stroke-linecap:square;stroke:#573a0d"/><path d="m8.5 30.9-2-1.2 0 6L17.3 42l0-2-1.7-1 0-2L13.5 35.8l0-4-5-2.9 0 2z" id="path3831" style="fill:#2e3436;stroke-linecap:round;stroke:#2e3436"/><path d="m6.9 36 3.5-2 3.5 2 1.6 0.9 0 2 2 1.2 0 2L6.9 36z" id="path3870" style="fill:#555753;stroke:#2e3436"/><path d="M26 19 31.5 22.2l0 2L38.1 28 41.6 26 45.5 23.7l0 2.5 0-4L32.9 15 26 19z" id="path3851" style="fill:#fce94f;stroke-linecap:round;stroke-linejoin:round;stroke:#625802"/><path d="m24.8 18.3 8.7-5 0 2-8.7 5 0-2z" id="path5684" style="fill:#e9b96e;stroke-dashoffset:0.5;stroke-linecap:round;stroke-linejoin:round;stroke:#573a0d"/><path d="M26 5 3.5 18 17.5 26.1 31.5 18.2 24.8 14.3 33.5 9.3 26 5z" id="path3821" transform="translate(0,4)" style="fill:#73d216;stroke-linecap:square;stroke-linejoin:round;stroke:#325b09"/><path d="m17.5 28.1 0 2 1.6 0.9 0 2 5.4 3.1 12-7.1 0-2.1 2-1.2 0-1.5-7-4-14 7.9z" id="path3825" transform="translate(0,4)" style="fill:#729fcf;stroke-linecap:round;stroke:#19314b"/><g id="g5691"><path id="path3862" d="m13.9 20 6.9 4-6.9 4-6.9-4 6.9-4z" class="s0"/><g id="g3858"><path d="m15.6 21 1.7 1 1.7-1 0-6-1.7-1-1.7 1 0 6z" id="path3833" transform="translate(-3.4641015,2)" style="fill:#c17d11;stroke-linecap:round;stroke:#8f5902"/><path d="M10 13.8 17.3 18l6.6-3.8 0-8.3L17.3 2 10 6.2l0 7.5z" id="path3837" transform="translate(-3.4641015,2)" class="s1"/></g></g><g id="g5686" transform="translate(-4.2591582e-7,2)"><path transform="translate(24.248712,-2)" d="m13.9 20 5.2 3-5.2 3-5.2-3 5.2-3z" id="path3868" class="s0"/><path d="M15.7 21.1 17.3 22l1.4-0.8 0-8.4L17.3 12l-1.6 1.1 0 8z" id="path3853" transform="translate(20.78461,0)" class="s1"/></g><path d="M12.1 33 11.3 32.5" id="path3872" style="fill:none;stroke-dasharray:0.5;stroke-dashoffset:0.3;stroke-width:0.5;stroke:#ef2929"/><path d="m45.5 26.7-4 2.3 0 1-2 1.2 0 2-2 1.2 0 4 8-4.6 0-7z" id="path3874" style="fill:#888a85;stroke-dashoffset:0.5;stroke-linecap:round;stroke:#2e3436"/></g></svg>
```

Comment:
lol, this shouldn't be changed

Reply:
whoops, i thought i left the files out of my git commit... fixed this, squashed the commits see below